zone/bot.cpp

```diff
@@ -6595,7 +6630,21 @@ bool Bot::DoFinishedSpellGroupTarget(uint16 spell_id, Mob* spellTarget, EQ::spel
 		);
 		SpellOnTarget(spell_id, this);
 		entity_list.AESpell(this, this, spell_id, true);
-	} else {
+	}
+	else if (raid) 
+	{
+		//for (auto& iter : raid->GetRaidGroupMembers(raid->GetGroup(this->GetName()))) {
+		std::vector<RaidMember> raid_group_members = raid->GetRaidGroupMembers(raid->GetGroup(this->GetName()));
```

Comment:
No need for `this`, unused comment above.

Reply:
Updated